printer_mods/README.md

```diff
@@ -27,6 +27,8 @@ like so:
 | --- | --- | --- | --- | --- | --- | --- | --- |
 |120decibell|[Side entry exhaust housing](./120decibell/exhaust_housing_side_entry)| Exhaust housing with dual cover screws and single or dual side entry filament paths. |:x:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
 ||[Huvud Chain Mount](./120decibell/huvud_chain_mount)| Huvud rear mount for Clockwork or LGX that supports a cable chain. |:x:|:heavy_check_mark:|:grey_question:|:x:|:heavy_check_mark:|
+|3dpandme|[FANnypack zero](./3dpandme/Fannypack_zero/)|Electronics cooling solution for Voron Zero|:heavy_check_mark:| :x: | :x: | :x: | :x:|
+| |[Thermistor holder zero](./3dpandme/Thermistor_holder_zero/)|Extended thermistor holder for Voron Zero|:heavy_check_mark:| :x: | :x: | :x: | :x:|
```

Comment:
techinically this mod will work on the trident and v2, but it's up to you if you want to check them off or not.

Reply:
Changes done as suggested :)